control-plane/rest/service/src/main.rs

```diff
@@ -30,6 +36,10 @@ pub(crate) struct CliArgs {
     #[clap(long)]
     http: Option<String>,
 
+    /// The bind address for the health REST server.
+    #[clap(long, default_value = "[::]:9091")]
+    health_endpoint: SocketAddr,
```

Comment:
might want to consider separating into ip and port

Reply:
Why not accept it together?

Reply:
This is no longer needed as the health service is not a separate endpoint anymore.